src/main/java/mekanism/common/content/gear/mekasuit/ModuleChargeDistributionUnit.java

```diff
@@ -97,17 +98,17 @@ private void chargeInventory(IEnergyContainer energyContainer, Player player) {
     }
 
     /** return rejects */
-    private FloatingLong charge(IEnergyContainer energyContainer, ItemStack stack, FloatingLong amount) {
-        if (!stack.isEmpty() && !amount.isZero()) {
+    private long charge(IEnergyContainer energyContainer, ItemStack stack, long amount) {
+        if (!stack.isEmpty() && amount != 0L) {
```

Comment:
Just to be safe and because it makes the intent clearer. Change this to `amount > 0`